crates/sncast/src/main.rs

```diff
@@ -122,9 +124,22 @@ fn main() -> Result<()> {
     let runtime = Runtime::new().expect("Failed to instantiate Runtime");
 
     if let Commands::Script(script) = cli.command {
+        let manifest_path = match cli.path_to_scarb_toml {
+            Some(path) => path,
+            None => get_scarb_manifest().context("Failed to obtain manifest path from Scarb")?,
+        };
```

Comment:
Could we create a function `load_manifest`? it would ensure error is always the same.

Reply:
This does not load it, but rather searches for it. I have added such function in https://github.com/foundry-rs/starknet-foundry/pull/1569 (`ensure_scarb_manifest_path`, and this function additionally verifies it exists. as suggested in this PR, the name is a subject to change). Is that enough, or did you have something else in mind as well?